index.html

```diff
@@ -10826,6 +10849,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 					</tr>
 				</tbody>
 			</table>
+			<p>The attribute's <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a> and <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> are both the Undefined state.</p>
```

Comment:
The `<tr>` for the `"undefined"` keyword is missing a `<th>` to name its state. In other words there should be a line above L10848 with `<th class="state-description" scope="row">Undefined</th>`. (I cannot comment on L10848 sadly).

I don't think we want the missing value default here to return the Undefined state, that will mean the `.ariaChecked` property will return `"undefined"` when `aria-checked` isn't on the element. I presume we want to return `null` instead? Browsers currently return null, so missing should be "no state", i.e. removed:

```suggestion
			<p>The attribute has no <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a> and its <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> are both the Undefined state.</p>
```

Reply:
Fixed in new PR: https://github.com/w3c/aria/pull/2484/files.